src/pages/account/mijn-mappen/[collectionSlug]/index.tsx

```diff
@@ -154,7 +162,9 @@ const AccountMyCollections: NextPage = () => {
 						iconStart={<Icon name="export" />}
 						onClick={(e) => {
 							e.stopPropagation();
+							save(metadataExport, `${activeCollection?.name || 'colection'}.xml`);
```

Comment:
use lodash kebab case to avoid special chars/spaces in filenames
```
kebabCase('Foo Bar');
// => 'foo-bar'
```
https://lodash.com/docs/4.17.15#kebabCase